src/libs/ReportActionsUtils.ts

```diff
@@ -144,12 +151,16 @@ function isModifiedExpenseAction(reportAction: OnyxEntry<ReportAction> | ReportA
     return reportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.MODIFIED_EXPENSE;
 }
 
+function getReportActionOriginalMessage<T>(reportAction: PartialReportAction) {
+    return (reportAction?.originalMessage ?? reportAction?.message?.[0] ?? reportAction?.message) as T;
```

Comment:
@c3024 Let see `getReportActionOriginalMessage` here, I'm returning `reportAction?.message` as the final fallback. So if we merge this PR and then the phase is complete the App will not be broken because at this time `reportAction.originalMessage` and `reportAction?.message?.[0]` will be undefined and then this function will return `reportAction?.message`.